R/plumber.R

```diff
@@ -402,6 +320,18 @@ plumber <- R6Class(
 
       private$mnts[[path]] <- router
     },
+    #' @description Unmount a plumber router
+    #' @param path a character string. Where to mount router.
+    unmount = function(path){
+      # Ensure that the path has both a leading and trailing slash.
+      if (!startsWith(path, "/")) {
+        path <- paste0("/", path)
+      }
+      if (!endsWith(path, "/")) {
+        path <- paste0(path, "/")
+      }
```

Comment:
```suggestion
      if (!grepl("^/", path)) {
        path <- paste0("/", path)
      }
      if (!grepl("/$", path)) {
        path <- paste0(path, "/")
      }
```

Reply:
Should we replace across package? They are currently being used in master.